extensions/poetry/fixtures/.phylum_project

```diff
@@ -0,0 +1,5 @@
+---
+id: 7f78e2f6-9833-40d1-95f7-432f8d48a963
+name: test-poetry
+created_at: "2022-07-27T15:21:23.503757423+02:00"
+group_name: ~
```

Comment:
Why add a `.phylum_project` for a fixture/test? Feels like that should just always be specified in the test as parameters. That way parameters are directly obvious when looking at a test case and there's no auxiliary files necessary.

Reply:
This was actually an oversight. It is not meant to be there.